packages/components/atoms/f-error-message/src/components/_tests/ErrorMessage.test.js

```diff
@@ -40,4 +40,15 @@ describe('ErrorMessage', () => {
         // Assert
         expect(content.element.innerHTML).toBe(propsData.slots.default);
     });
+
+    it('should contain the `aria-live="assertive"` attribute so screen readers can read back errors correctly', () => {
+        // Arrange
+        const wrapper = shallowMount(ErrorMessage, propsData);
+
+        // Act
+        const content = wrapper.find('[data-test-id="content"]');
+
+        // Assert
+        expect(content.attributes('aria-live')).toMatchSnapshot();
```

Comment:
Would be the check for correct attribute more effective with:
```suggestion
        expect(content.attributes('aria-live')).toBe('assertive');
```

Reply:
Done.